app/views/shared/_logo_design.html.erb

```diff
@@ -1,7 +1,12 @@
 <div class="grid gap-6 m:grid-cols-2">
   <div class="grid gap-6">
-    <% account.bg_color_hex = user_colors(account)[:bg] if account.bg_color_hex.blank? %>
-    <% account.text_color_hex = user_colors(account)[:text] if account.text_color_hex.blank? %>
+    <% if account.class.name == "User" %>
+      <% account.setting.brand_color1 = user_colors(account)[:bg] if account.setting && account.setting.brand_color1.blank? %>
+      <% account.setting.brand_color2 = user_colors(account)[:text] if account.setting && account.setting.brand_color2.blank? %>
+    <% else %>
+      <% account.bg_color_hex = user_colors(account)[:bg] if account.bg_color_hex.blank? %>
```

Comment:
We should probably try to keep the names consistent between `User` and `Organization`.

Reply:
Yeah thats less than ideal. Given we have the `config_` attributes and these I could see a naming refactor PR to update those columns in the database.

Reply:
I agree. I discussed the current approach in this view with Ridhwana, and it is confusing.